reg/drone/service/gnss/FixStatus.0.1.uavcan

```diff
@@ -0,0 +1,13 @@
+# Status of the GNSS navigation solution. This information is needed for indication and general status monitoring only.
+# Data consumers should not use this info and rely on the error estimates reported via covariance matrices instead.
+# If the sensor is equipped with other means of deriving the navigation solution (dead reckoning, VIO, UWB, etc.),
+# then this value may not be representative of the actual status of the solution (only matrices are).
+
+uint2 NONE = 0
+uint2 TIME = 1  # Only time is available.
+uint2 D2   = 2  # The time, latitude, longitude, and NE-plane velocity are available.
```

Comment:
Is there any specific reason for having this state abbreviated by `D2`? My issue with this shorthand is that I can't infer what it actually means and would rather have it spelled out in long form, i.e. `TIME_POSITION_VELOCITY`. In a similar way `D3` could be `ALL` (or an enumeration as `TIME_POSITION_VELOCITY_HEADING_...`).

Reply:
It is supposed to say "2D" and "3D", but [a DSDL identifier cannot start with a digit](https://github.com/UAVCAN/specification/blob/7cacf58936a494de973c81bfe13f1ecbcb71337c/specification/dsdl/grammar.parsimonious#L7). We can change it as you suggested, but upon further consideration, I am actually inclined to replace this with a simple boolean flag indicating whether the fix is attained at all. The available data can be determined from covariance matrices. This change would remove one type from the service definition making it (marginally) simpler.

Reply:
I agree with your change in 669d7166be579fb8d092ef91caf050f47b667160 ✔️, it is probably a sisyphean task to try to find `FixStatus` enumeration values to suit the requirements of all available GNSS sensors. A boolean flag + already existing covariance matrices should achieve the same result while eliminating redundant information.